lib/glific/contacts/import.ex

```diff
@@ -53,20 +53,28 @@ defmodule Glific.Contacts.Import do
     end
   end
 
-  defp process_data(contact, group_id) do
-    result =
-      case contact.optin_time do
-        nil -> BSPContacts.Contact.create_or_update_contact(Map.put(contact, :method, "Import"))
-        _ -> Contacts.optin_contact(Map.put(contact, :method, "Import"))
-      end
+  defp process_data(contact_attrs, group_id) do
+    {:ok, contact} = Contacts.maybe_create_contact(contact_attrs)
 
-    case result do
-      {:ok, contact} ->
-        add_contact_to_group(contact, group_id)
-        contact
+    if group_id do
```

Comment:
Adding this as a different code so that we can add the row level collection easily in the future.